lib/signaling/v2/peerconnection.js

```diff
@@ -1317,4 +1353,50 @@ function updateRecycledTransceivers(pcv2) {
   });
 }
 
+/**
+ * Perform certain updates after an SDP negotiation is completed.
+ * @param {PeerConnectionV2} pcv2
+ * @returns {void}
+ */
+function negotiationCompleted(pcv2) {
```

Comment:
@makarandp0 Can we enhance this function to:
```js
function negotiationCompleted(pcv2) {
  if (pcv2._isUnifiedPlan) {
    updateRecycledTransceivers(this);
    updateLocalCodecs(this);
    updateRemoteCodecMaps(this);
  }
  if (pcv2._isRTCRtpSenderParamsSupported) {
    updateEncodingParameters(pcv2);
  }
}
```
?

Reply:
thanks @manjeshbhargav, That looks much better.